pkg/build/stage-ibmcos.go

```diff
@@ -36,6 +37,7 @@ func NewIBMCOSStager(stageLocation, repoRoot, targetBuildArch string) (*IBMCOSSt
 		RepoRoot:        repoRoot,
 		Region:          matches[2],
 		Bucket:          matches[3],
+		Directory:       matches[4],
```

Comment:
Path is the more relevant word here because this may contain multiple levels of directory.